README.md

```diff
@@ -1,24 +1,72 @@
-# README
+# Budgeta
 
```

Comment:
- [ ] [OPTIONAL] Let's also add the screenshot of your app also 📷 to the readme file 📃.